activities/service/v1/notifications.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.v1";
+option go_package = "activities.v1";
+
+import "google/protobuf/empty.proto";
+import "activities/domain/v1/domain.proto";
+
+service notificationsService {
+  rpc updateNotificationChannelsConfiguration (notificationChannelConfiguration) returns (google.protobuf.Empty);
+  rpc getNotificationChannelsConfiguration (google.protobuf.Empty) returns (notificationChannelConfiguration);
+}
+
+message notificationChannelConfiguration{
+  bool notify_by_email = 1;
```

Comment:
I think this is really too simple. We should have different types of notifications. For example
* email
* webhook
* sms
* ...

And we should be able to have multiple notifications configured, not just one.

Reply:
what you mean? have a type instead of the bool? that means we need to send multiple config in order to have multiple notifications channels?
my idea was to have all the types that we allow as a bool/config (email/webhook) so we can notify through all of them. eg if you have `notify_by_email: true` and `webhook_url: something.com` you will receive on both.